commit d8e00c75c203c8972942ef8b5930c0ba5214a72f
Author: Todd Whiteman <todd.whiteman@joyent.com>
Date:   2019-06-04T13:19:58-07:00 (4 months ago)
    
    TRITON-1722 docker volume creation failure test is failing in nightly

diff --git a/test/integration/cli-nfs-shared-volumes-create-failure.test.js b/test/integration/cli-nfs-shared-volumes-create-failure.test.js
index e3fe90f..9d4a547 100644
--- a/test/integration/cli-nfs-shared-volumes-create-failure.test.js
+++ b/test/integration/cli-nfs-shared-volumes-create-failure.test.js
@@ -5,7 +5,7 @@
  */
 
 /*
- * Copyright (c) 2017, Joyent, Inc.
+ * Copyright (c) 2019, Joyent, Inc.
  */
 
 //
@@ -107,21 +107,14 @@ test('Volume creation should fail when provision fails', function (tt) {
                 size: '10G',
                 name: testVolumeName
             }, function volumeCreated(err, stdout, stderr) {
-                var expectedErr = 'Error response from daemon: (InternalError) '
-                    + 'volume creation failed';
-                var matches;
-
-                // Make a RegExp from the expectedErr but we need to escape the
-                // '(' and ')' characters to '\(' and '\)' so that the regex
-                // will not treat that as a grouping.
-                var re = new RegExp(expectedErr.replace(/[()]/g, '\\$&'));
-
-                matches = stderr.match(re);
+                var expectedErr = 'Error response from daemon: '
+                    + 'problem creating volume';
+                var matches = stderr.match(expectedErr);
 
                 t.ok(err, 'volume creation should not succeed');
                 // with this, we get the actual error message if it fails
                 t.equal((matches ? matches[0] : stderr), expectedErr,
-                    'expected InternalError');
+                    'expected "problem creating volume" error message');
 
                 t.end();
             });
